### PR TITLE
Update to 5.15-21.08 runtime + deps (take 2)

### PR DESCRIPTION
**Pull request by ozmartian**

### CODE REVIEW

**Comment by ozmartian** [comment]
Issue logged at:

https://github.com/flathub/flathub/issues/3082

**Comment by hfiguiere** [comment]
The icons are in `share/icons/icons/hicolor/128x128/apps/com.ozmartians.VidCutter.png` which is not the right path.

File: com.ozmartians.VidCutter.json [hunk 1]
```diff
@@ -474,17 +474,16 @@
...
             "mkdir -p ${FLATPAK_DEST}/share/fonts",
+            "mkdir -p ${FLATPAK_DEST}/share/icons",
             "cp vidcutter/fonts/* ${FLATPAK_DEST}/share/fonts",
-            "cp -R data/icons/ ${FLATPAK_DEST}/share/icons",
-            "mkdir -p ${FLATPAK_DEST}/share/applications",
-            "cp data/desktop/com.ozmartians.VidCutter.desktop ${FLATPAK_DEST}/share/applications",
-            "mkdir -p ${FLATPAK_DEST}/usr",
-            "ln -sf ${FLATPAK_DEST}/share ${FLATPAK_DEST}/usr"
+            "cp -R data/icons/hicolor ${FLATPAK_DEST}/share/icons",
+            "cp data/desktop/com.ozmartians.VidCutter.desktop ${FLATPAK_DEST}/share/applications"
```
Comment (hfiguiere) [inline com.ozmartians.VidCutter.json:482]:
```suggestion
            "cp -R data/icons/hicolor ${FLATPAK_DEST}/share/icons/",
            "install -Dm644 -t ${FLATPAK_DEST}/share/applications data/desktop/com.ozmartians.VidCutter.desktop"
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build

**Comment by hfiguiere** [comment]
builds where cancelled so we don't actually know if your last changes did work.

File: com.ozmartians.VidCutter.json (hunk 1 above)
Comment (hfiguiere) [reply com.ozmartians.VidCutter.json:482]: I don't think these are neccessary tbf. Let's see how the current build do. (all previous were cancelled)

**Comment by ozmartian** [comment]
> builds where cancelled so we don't actually know if your last changes did work.

**Comment by hfiguiere** [comment]
I think you also need this change:

https://github.com/flathub/com.ozmartians.VidCutter/pull/28/commits/91b2007c0dc0c37345800255b68311b5b60e793b#diff-21095ac5afedb596a89240f2348545ba529031a43ff7ebab366c4742aca743d1R90

**Pull request closed without merging**